lib/i18n/js.rb

```diff
@@ -183,7 +183,7 @@ def self.translations
         #
         # So the input is wrapped by our class for better `#slice`
         Private::HashWithSymbolKeys.new(translations).
-          slice(*::I18n.available_locales).
+          slice(*::I18n::JS.js_available_locales).
```

Comment:
I think that better to use `js_available_locales` name for new method, because when I used:

```
slice(*available_locales).
```

I got some mystic errors, until I found that we use `self.backend.instance_eval do` here.

So, maybe the other name of method will help to support gem with lilttle less pain in the future :)

Reply:
I agree with your naming which matches its purpose.